src/olympia/scanners/admin.py

```diff
@@ -697,6 +700,51 @@ class ScannerQueryResultAdmin(
         ('version__files__is_signed', FileIsSigned),
     )
 
+    ordering = ('version__addon_id', 'version__channel', 'version__created')
```

Comment:
We don't have an index on `Version` doing that, it might help this and the `AddonVersionViewSet`, I'll investigate.

Reply:
I've tested it on prod and played with it locally and it looks like this wouldn't help, at least with the version of MySQL we're using (the table that would need better indexes is `scanners_scannerquerymatch`, we'd need some serious denormalization for this to work). It's slow when there are several hundred thousands results (worst case) but should still be bearable.

Reply:
Also, because the problem is the ordering of results with `scanners_scannerquerymatch` this actually doesn't seem to make it any worse than the current ordering on `created`.